### PR TITLE
Nano: Fix keras model.compile

### PR DESCRIPTION
**Pull request by MeouSker77**
## Description

If a model has called `model.compile()`, then after trace or quantize, it will lose its loss function and metrics and have to call `compile` once again, this PR fix this issue.

### CODE REVIEW

**Review by rnwang04 — APPROVED** [review]
LGTM
